cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -304,6 +305,17 @@ func (c *CVCController) getVolumePolicy(
 	return volumePolicy, nil
 }
 
+// isReplicaAffinityEnabled checks if replicaAffinity has been enabled using
+// cstor volume policy
+func (c *CVCController) isReplicaAffinityEnabled(cvc *apis.CStorVolumeClaim) bool {
+	policyName := cvc.Annotations[string(apis.VolumePolicyKey)]
+	volumePolicy, err := c.getVolumePolicy(policyName, cvc)
```

Comment:
Let's return default policy if policy is not provided by user in storage class

Reply:
Default policy is not part of this PR changes